src/Ng/CommandHelpers.cs

```diff
@@ -179,7 +182,8 @@ public static StorageFactory CreateSuffixedStorageFactory(string suffix, IDictio
                 { Arguments.StorageKeyValue, Arguments.StorageKeyValue + suffix },
                 { Arguments.StorageContainer, Arguments.StorageContainer + suffix },
                 { Arguments.StoragePath, Arguments.StoragePath + suffix },
-                { Arguments.StorageSuffix, Arguments.StorageSuffix + suffix }
+                { Arguments.StorageSuffix, Arguments.StorageSuffix + suffix },
+                { Arguments.StorageOperationMaxExecutionTimeInSeconds, Arguments.StorageOperationMaxExecutionTimeInSeconds + suffix }
```

Comment:
nit: put a comma at the end of the last item in the dictionary initializer to avoid future diff noise (note it looks like there are two lines changes here instead of just one).